content/organizing-faq.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Kubernetes Community Day FAQ 
+---
+
+# Are Kubernetes Community Days limited to Kubernetes?
+
+No! Any open source, cloud native technology is appropriate to cover at a KCD. The only requirements are that presentations not be vendor pitches. Please see the cloud native [definition](https://github.com/cncf/toc/blob/master/DEFINITION.md).
```

Comment:
Since the name must include Kubernetes (see below), then can be any combination of Kubernetes+any other CNCF project, provided that the definition of "cloud native technology" coincides with "any project accepted by CNCF"